src/heatmaps.ts

```diff
@@ -6,9 +6,13 @@ import { PostHog } from './posthog-core'
 import { document, window } from './utils/globals'
 import { getParentElement, isTag } from './autocapture-utils'
 import { HEATMAPS_ENABLED_SERVER_SIDE, TOOLBAR_ID } from './constants'
-import { isUndefined } from './utils/type-utils'
+import { isEmptyObject, isObject, isUndefined } from './utils/type-utils'
 import { logger } from './utils/logger'
 
+const DEFAULT_FLUSH_INTERVAL = 5000
+const HEATMAPS = 'heatmaps'
+const LOGGER_PREFIX = '[' + HEATMAPS + ']'
```

Comment:
Why the separate const here?

Reply:
force of habit :)
as the number of log messages grows by copy pasta, the file size doesn't increase by the multiple of the string bytes